tests/diesel/src/models.rs

```diff
@@ -0,0 +1,18 @@
+use compact_str::CompactString;
+use diesel::prelude::*;
+
+#[derive(Queryable, Selectable)]
+#[diesel(table_name = crate::schema::todos)]
+#[diesel(check_for_backend(diesel::sqlite::Sqlite))]
+pub struct Todo {
+    pub id: i32,
+    pub title: CompactString,
+    pub done: bool,
+}
+
+#[derive(Insertable)]
+#[diesel(table_name = crate::schema::todos)]
+pub struct NewTodo<'a> {
+    pub title: &'a str,
```

Comment:
Why not make this

```suggestion
    pub title: CompactString,
```

to test serialization?

Reply:
Ah, yes, you're right!